classes/class-apm-admin-settings.php

```diff
@@ -381,7 +415,9 @@ public static function save() {
 		public static function mautic_authenticate_update() {
 
 			if ( isset( $_POST['ampw-save-authenticate'] ) && 'Save and Authenticate' == esc_attr( $_POST['ampw-save-authenticate'] ) ) {
-				AP_Mautic_Api::authenticate_update();
+				$post_data = $_POST;
```

Comment:
Should `$_POST` be sanitized here? or this is handled in the `$instance->connect()`?